### ISSUE TITLE
[Feature request]: Support familiar faces on Homebase 3

### ISSUE THREAD

**Issue by Palmke** [bropat/eufy-security-client#238]
### Describe the solution you'd like

In the past I added the familiar faces to the doorbell push data.
I have taken a quick look at the code, but I'm not sure why this logic isn't working for the new Homebase 3. 

The push data is correctly parsed, but I don't get a `person detected` event:
```
{"type":"event","event":{"source":"device","event":"property changed","serialNumber":"T8160PZZZZZZZZZ","name":"personName","value":"<Name>"}}
{"type":"event","event":{"source":"device","event":"property changed","serialNumber":"T8160PZZZZZZZZZ","name":"identityPersonDetected","value":true}}
{"type":"event","event":{"source":"device","event":"property changed","serialNumber":"T8160PZZZZZZZZZ","name":"personName","value":""}}
{"type":"event","event":{"source":"device","event":"property changed","serialNumber":"T8160PZZZZZZZZZ","name":"identityPersonDetected","value":false}}
````
Perhaps is has to do with that in this case the event type from the notification is not `personDetected`, but rather `identityPersonDetected` and `strangerPersonDetected`.
Push event type is `event_type: 3111,`, so that would explain it.

This again is an issue for administrative purposes.
I tend to pick this up myself.

## Is there demand for splitting familiar and stranger identities in separate events?

**Comment by bropat** [comment]
> Is there demand for splitting familiar and stranger identities in separate events?

I don't think so.

**Comment by bropat** [comment]
Implemented in version `2.2.1`.
